### DISCUSSION TITLE
How to do a refund?

### DISCUSSION THREAD

**Discussion by gopeter** [craftcms/commerce#3441] [Q&A]
I am feeling totally dumb but I think I just overlooked something. I've read a lost about the possibility to do partial refunds, but I don't get how to do this.

If a user has bought 3 items and wants me to refund 2 of them, how can I achieve this in the backend? I know that I can refund a specific amount in the transactions tab of the order, but it sounds (in the documentation or some issues and discussions) like I'm able to select the items I want to refund somewhere.

<img width="1514" alt="image" src="https://github.com/craftcms/commerce/assets/1242960/e451f012-1ca3-47fa-a402-97328fb2807f">

**Comment by lukeholder** [comment]
This is not possible at the moment. We hope to add a refund and returns feature in the future. For now in Commerce 5 you can mark the item as unfulfilled in the fulfillment modal, but the refund is not associated with the item. Thanks for the feedback.
